photo.go

```diff
@@ -9,9 +9,17 @@ import (
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/s3"
+
+	"github.com/globocom/gothumbor"
 )
 
-type ImgixPhoto struct {
+type ImgixRescaledPhoto struct {
+	Key     string
```

Comment:
looking for cleanest way to move these common fields out... it's a `struct` after all so 'tis strange. halp?

Reply:
If by move out you mean in OOP language create a parent class with these common attributes, you could use structure embedding. Look here: https://golang.org/doc/effective_go.html#embedding